standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1823,8 +1823,46 @@ public enum ConfVars {
     ),
     PROPERTIES_SERVLET_AUTH("hive.metastore.properties.servlet.auth",
         "hive.metastore.properties.servlet.auth", "jwt",
+            new StringSetValidator("simple", "jwt"),
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    ICEBERG_CATALOG_SERVLET_FACTORY("hive.metastore.catalog.servlet.factory",
+            "hive.metastore.catalog.servlet.factory",
+            "org.apache.iceberg.rest.HMSCatalogFactory",
+            "HMS Iceberg Catalog servlet factory class name."
+            + "The factory needs to expose a method: "
+            + "public static HttpServlet createServlet(Configuration configuration);"
+    ),
+    ICEBERG_CATALOG_SERVLET_PATH("hive.metastore.catalog.servlet.path",
+        "hive.metastore.catalog.servlet.path", "iceberg",
+        "HMS Iceberg Catalog servlet path component of URL endpoint."
+    ),
+    ICEBERG_CATALOG_SERVLET_PORT("hive.metastore.catalog.servlet.port",
+        "hive.metastore.catalog.servlet.port", -1,
+        "HMS Iceberg Catalog servlet server port. Negative value disables the servlet," +
+            " 0 will let the system determine the catalog server port," +
+            " positive value will be used as-is."
+    ),
+    ICEBERG_CATALOG_SERVLET_AUTH("hive.metastore.catalog.servlet.auth",
+        "hive.metastore.catalog.servlet.auth", "jwt",
```

Comment:
should it be oauth2?

Reply:
If I'm not mistaken, the Iceberg rest spec allows using a Bearer token for authentication which is what our 'jwt' auth does.

Reply:
you might be right
https://github.com/apache/iceberg/blob/be9808fb3d75b1697fc4ed45c02602de9cfb6371/core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Util.java#L95
````
JWT_TOKEN_TYPE = "urn:ietf:params:oauth:token-type:jwt";
````

Reply:
[testCatalogBearerTokenWithClientCredential](https://github.com/apache/iceberg/blob/apache-iceberg-1.6.1/core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java#L461-L484)